README.md

```diff
@@ -101,33 +87,41 @@ parameter, the `key` as string and should return a string.
 
 - `serializedKey` (Optional) `{string | ToSerializedKeyStrategy}` - Used as the
   key in the serialized object
-- `toJSONStrategy` (Optional) `{ToJSONStrategy}` - Used when serializing
-- `fromJSONStrategy` (Optional) `{FromJSONStrategy}` - Used when deserializing
+- `toJSONStrategy` (Optional) `{Strategy}` - Used when serializing
+- `fromJSONStrategy` (Optional) `{Strategy}` - Used when deserializing
```

Comment:
I think this needs to be reverted back to `ToJSONStrategy` and `FromJSONStrategy`